metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/EdtfValidator.java

```diff
@@ -111,6 +114,7 @@ private static boolean validateInstant(InstantEdtfDate instantEdtfDate, boolean
     if (validateDatePart(edtfDatePart)) {
       EdtfTimePart edtfTimePart = instantEdtfDate.getEdtfTimePart();
       if (validateTimePart(edtfTimePart)) {
+
         // TODO: 20/07/2022 Does this mean that if it's not standalone, it is then allowed to have both null/unknown??
```

Comment:
For non-standalone(interval dates) this check is not required since the date can be empty(which is considered valid).